### ISSUE TITLE
Bookworm does not correctly initialize a specific bulgarian SAPI5 TTS voice

### ISSUE THREAD

**Issue by k-kolev1985** [mush42/bookworm#31]
## Describe the Problem

A friend of mine has reported this to me and I was able to reproduce it.

There is one very old (made in 2005 I think) SAPI5 bulgarian voice. It is called "Gergana" (the software product itself is called "SpeechLab").

When the user configures Bookworm to use this voice and presses F5 to read an opened book with it, the voice just stutters on the first syllable from the text.

I've restarted Bookworm with debugging enabled and reproduced the problem. Then Ive gathered the log files from the \AppData\Roaming\Bookworm\logs\ folder. I'll attach them here. If needs be, I'll personally send you the voice for you to test with it and some bulgarian books.

I'm afraid however, that this is an incompatibility between the voice "Gergana" and the method which Bookworm uses to access the SAPI5 TTS voices in Windows and I don't know if you'll be able to do something about it. Still, I'm reporting it, just in case I'm wrong and the problem can be fixed.

## To Reproduce

1. Install the bulgarian SAPI5 voice "Gergana" (the product "SpeechLab").
2. Open Bookworm.
3. Configure Bookworm to use the SAPI5 voice "Gergana".
4. Open a bulgarian book in Bookworm.
5. Press F5 to start reading the book with Bookworm with the voice "Gergana".

## Expected behavior

The voice should start reading the book.

## The File You've Opened

It is an EPUB book in bulgarian (not a particular one).

## Desktop (please complete the following information):

 - Operating system: Windows 10 Pro version 1909 (build 18363.535), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
 - Bookworm version: 0.1b4, 32-bit, in bulgarian.

## Additional context

1. The voice "Gergana" is a 32-bit only voice. That means, that 64-bit programs that under 64-bit Windows look only for 64-bit voices, will not be able to use this voice.
2. The problem is reproducible under 32-bit Windows 7 as well.
[Bookworm_Logs_2019-12-17.zip](https://github.com/mush42/bookworm/files/3973013/Bookworm_Logs_2019-12-17.zip)

**Comment by DraganRatkovich** [comment]
@k-kolev1985 Can you send me a voice? I want to test it with the latest build of Bookworm.

**Comment by k-kolev1985** [comment]
@DraganRatkovich 
Yes, I think I can. But you have to be aware, that it is a 32-bit only voice and 64-bit versions of programs usually do not detect and therefore cannot use it. That includes the 64-bit version of Bookworm. The 32-bit version of Bookworm should be able to detect it.

BTW: There is a newer bulgarian SAPI5 TTS, that also does not for some reason work with Bookworm. But unlike the old "SpeechLab" one, the new one has restrictions on distribution and I'm not allowed to distribute/share it, at least for the time being. I can however file a ticket here and provide logs, if that will help to diagnose the issue in this case.

**Comment by k-kolev1985** [comment]
@DraganRatkovich 
How can I contact you privately? I cannot share this software publicly - its license does not allow it.
